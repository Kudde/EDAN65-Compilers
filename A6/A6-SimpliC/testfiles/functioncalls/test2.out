Fun: bakeBrownies
Reach: { eatIceCream brewCoffee }
 > eatIceCream

Fun: eatIceCream
Reach: { brewCoffee }
 > brewCoffee

Fun: brewCoffee
Reach: { }

Fun: main
Reach: { eatIceCream brewCoffee bakeBrownies }
 > eatIceCream
 > bakeBrownies

