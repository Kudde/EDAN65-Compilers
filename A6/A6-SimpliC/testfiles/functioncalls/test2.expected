Fun: bakeBrownies
Reach: { brewCoffee eatIceCream }
 > eatIceCream

Fun: eatIceCream
Reach: { brewCoffee }
 > brewCoffee

Fun: brewCoffee
Reach: { }

Fun: main
Reach: { bakeBrownies brewCoffee eatIceCream }
 > eatIceCream
 > bakeBrownies
